transform/mattermost-analytics/models/staging/stripe/stg_stripe__customers.sql

```diff
@@ -18,8 +18,10 @@ customers as (
         invoice_prefix,
         invoice_settings,
         livemode,
-        metadata:"contactfirstname"::varchar as contact_first_name,
-        metadata:"contactlastname"::varchar as contact_last_name,
+        TRIM(COALESCE(metadata:"contactfirstname"::varchar, 
```

Comment:
These two columns need to be added to models file.

Reply:
They are part of 1 field, should they still be added? 

I don't think we can add the fields we are parsing to the models file, only the final field.

Reply:
Sorry, I don't understand what "final" field means. 

In any case, model `stg_stripe__customers` will include `contact_first_name` and `contact_last_name` columns. They must be added in `_stripe__models.yml`, under `stg_stripe__customers`  model's column definition.